uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -484,6 +480,35 @@ define([
       }, true);
     }
 
+    /**
+     * Watches title field to toggle Settings section locking
+     */
+    function watchTitleFieldIsFilledInProperly () {
+      var titleField = vm.fieldsIndexed.title;
+
+      $scope.$watch(function () {
+        return titleField.value;
+      }, function (title) {
+        var allowedToMoveToSettingsSection = !titleField.error && title !== '';
+
+        setAvailabilityOfFollowingSections(allowedToMoveToSettingsSection);
+      });
+    }
+
+    /**
+     * Sets the availability of the sections that follow the active section.
+     * Also sets the availability of the "Next section" button for the active section.
+     *
+     * @param {Boolean} enabled
+     */
+    function setAvailabilityOfFollowingSections (enabled) {
```

Comment:
can we change the name of this param to `isDisabled` ? Then we can remove the `!enabled` from here and just use `isDisabled`. Obviously you'd need to change the call to this function.